### DISCUSSION TITLE
How to use cut_slc tool

### DISCUSSION THREAD

**Discussion by sunzhe-96** [gmtsar/gmtsar#338] [General]
Hello, everyone, I want to know if 'cut_slc' tool should be used before 'preproc_batch_tops.csh data.in dem.grd 2'operation? I want to cut the image to reduce the study area, how should I use 'cut_slc' tool for that?

**Comment by Xiaohua-Eric-Xu** [comment]
You'll need to perform afterwards. If you specify region_cut in batch_tops.config, intf_tops.csh will cut it for you.

**Reply by sunzhe-96** [reply]
> You'll need to perform afterwards. If you specify region_cut in batch_tops.config, intf_tops.csh will cut it for you.

Thanks Eric,I'll try.
